starter/README.md

```diff
@@ -6,7 +6,114 @@ This is the project template for Stark applications.
 
 ## Starter structure
 
-TODO doc; see #32
+```txt
+|
++---config
+|   |
+|   |   webpack-custom-config.dev.json  # webpack configuration for the development environment
+|
++---src
+|   |
+|   +---app
+|   |   |
+|   |   +---foo                         # smart component folder
+|   |   |   |                           # create a folder for each component
+|   |   |   |   foo.component.css       # styles for this smart component
+|   |   |   |   foo.component.html      # template for this smart component
+|   |   |   |   foo.component.spec.ts   # jasmine test  for this smart component
+|   |   |   |   foo.component.ts        # configuration metadata for this smart component
+|   |   |   |   foo.e2e.ts              # end-to-end test for this smart component
+|   |   |   |   index.ts                # exports the smart component
+|   |   |   |
+|   |   |   app.component.css           # application base styles
+|   |   |   app.component.html          # application template
+|   |   |   app.component.spec.ts       # jasmine test for the app component
+|   |   |   app.component.ts            # app root component and controller
+|   |   |   app.e2e.ts                  # end-to-end test for the app class
+|   |   |   app.module.ts               # bootstraps code of the application (configures Angular module, ...)
+|   |   |   app.routes.ts               # routing configuration
+|   |   |   app.services.ts             # exports the application state
+|   |   |   index.ts                    # exports the app module
```

Comment:
same here:
Shouldn't we use the same way to do the barrel than the one we have in the framework ?
Instead of having index.ts in each folder, having foo.ts file beside foo folder.